manifests/m/Microsoft/ODBCDriver17forSQL/17.10.5.1/Microsoft.ODBCDriver17forSQL.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created with komac v2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.ODBCDriver17forSQL
+PackageVersion: 17.10.5.1
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.microsoft.com/download/6/f/f/6ffefc73-39ab-4cc0-bb7c-4093d64c2669/en-US/17.10.5.1/x64/msodbcsql.msi

```

Comment:
```suggestion
  InstallerType: msi
  InstallerUrl: https://download.microsoft.com/download/6/f/f/6ffefc73-39ab-4cc0-bb7c-4093d64c2669/en-US/17.10.5.1/x64/msodbcsql.msi
```

Reply:
This might be what's needed here, but I'm not completely sure that this is correct.